### PR TITLE
Rename artifacts for Azure Monitor Exporter

### PR DESCRIPTION
**Pull request by srnagar**
This PR renames the artifact for Azure Monitor Exporter based on the [arch board review](https://github.com/Azure/azure-sdk/issues/1975). The new artifact will be in `com.azure` groupId and the name of the artifact is `azure-opentelemetry-exporter-azuremonitor`.

### CODE REVIEW

File: sdk/monitor/azure-opentelemetry-exporter-azuremonitor/README.md [hunk 1]
```diff
@@ -61,8 +61,22 @@ System.out.println(resultCode.isSuccess());
 ```
 
 ## Key concepts
-For more information on the OpenTelemetry project, please review the [OpenTelemetry Specifications
-][opentelemetry_specification].
+
+Some of the key concepts for the Azure monitor exporter include:
```
Comment (alzimmermsft) [inline sdk/monitor/azure-opentelemetry-exporter-azuremonitor/README.md:65]: The product name is `Azure Monitor` right? So, should this be:

```suggestion
Some of the key concepts for the Azure Monitor exporter include:
```

File: sdk/monitor/azure-opentelemetry-exporter-azuremonitor/README.md [hunk 2]
```diff
@@ -114,5 +128,11 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 [opentelemetry_io]: https://opentelemetry.io/ 
 [span_data]: https://opentelemetry.lightstep.com/spans
 [sample_readme]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/monitor
-
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2Fsdk%monitor%2Fmicrosoft-opentelemetry-exporter-azuremonitor%2FREADME.png)
+[opentelemtry_spec]: https://opentelemetry.io/
+[instrumentation_library]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md#instrumentation-libraries
+[tracer_provider]: https://opentelemetry-python.readthedocs.io/en/stable/api/trace.html?highlight=TracerProvider#opentelemetry.trace.TracerProvider
+[span_processor]: https://opentelemetry-python.readthedocs.io/en/stable/_modules/opentelemetry/sdk/trace.html?highlight=SpanProcessor#
```
Comment (alzimmermsft) [inline sdk/monitor/azure-opentelemetry-exporter-azuremonitor/README.md:134]: Are there non-`en` links available for this?

File: sdk/monitor/azure-opentelemetry-exporter-azuremonitor/README.md (hunk 1 above)
Comment (srnagar) [reply sdk/monitor/azure-opentelemetry-exporter-azuremonitor/README.md]: Fixed.

File: sdk/monitor/azure-opentelemetry-exporter-azuremonitor/README.md (hunk 2 above)
Comment (srnagar) [reply sdk/monitor/azure-opentelemetry-exporter-azuremonitor/README.md]: Fixed.

**Review by samvaity — APPROVED** [review]

**Review by trask — APPROVED** [review]

**Pull request merged**